solvebio/contrib/streamlit/app_example.py

```diff
@@ -1,7 +1,13 @@
 import streamlit as st
 import pandas as pd
 
+from dotenv import load_dotenv
+
+# Load environment variables from .env file
+load_dotenv()
```

Comment:
Maybe to remove the dotenv from requirements as well?
https://github.com/solvebio/solvebio-python/blob/master/solvebio/contrib/streamlit/dev-requirements.txt#L4

Reply:
Yea we can, I figured it in the dev requirements so not a big deal either way.